auto/find-auto-dependa.py

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env python3
+# /// script
```

Comment:
what handles such things?  might be worth adding a comment above. 

Ideally should be some way `pip` could understand directly somehow... I bet someone thought of extending pip with support for parsing some header out, did they?

Reply:
What about prepending all scripts which oriented toward operation with github with `gh-` prefix?

Reply:
This is [inline script metadata](https://packaging.python.org/en/latest/specifications/inline-script-metadata/) from PEP 723.  The only programs I know of that support it are pipx, pip-run, and hatch.

Reply:
> What about prepending all scripts which oriented toward operation with github with gh- prefix?

I don't see a need for that.